src/components/Search/Search.module.scss

```diff
@@ -10,10 +10,10 @@ $clr-accent: #4183d7;
 }
 
 .input {
-  //background-color: $clr-accent;
+  position: relative;
```

Comment:
I think the background-color would need to be set to white here, since the color of background page will be apparent due to the new styling below

Reply:
Yeah , just added those changes

Reply:
![Screenshot (211)](https://user-images.githubusercontent.com/59141533/94988068-2f2b7380-0588-11eb-8421-aaf357e3e24f.png)